tests/integ/test_xgboost.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nitpick: replace `Copyright xxxx-xxxx Amazon.com, Inc. or its affiliates. All Rights Reserved.` with `Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.`. See #2532 and #2531.